_data/projects/cdnjs.yml

```diff
@@ -0,0 +1,14 @@
+name: cdnjs 
+desc: The most famous free and open source web front-end libraries cdn
+site: https://cdnjs.com
+tags:
+- cdn
+- js
+- css
+- libraries
+- web
+- frontend
+- javascript
+upforgrabs:
+  name: issues
+  link:　https://github.com/cdnjs/cdnjs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22wait+for+response%22+-label%3A%22in+progress%22+
```

Comment:
Seems like something with parsing this URL is causing the YAML parser to freak out.

Could this be simplified to `https://github.com/cdnjs/cdnjs/labels/Help%20wanted`, and then use `Help wanted` for the `name` above?